docs/_doc/Private-network.md

```diff
@@ -1,26 +1,74 @@
 ---
 title: Private network
 ---
-An Ethereum network is a private network if the nodes are not connected to the main
-network nodes. In this context private only means reserved or isolated, rather than
+
```

Comment:
I moved a section from the page above into this page as they both dealt with the whole bootnode process. I wonder if this was more the "advanced" steps that @fjl mentioned, so left alone for now. Maybe it isn't, but it's two different approaches.

I have done minimal editing to this page so far.